shared/src/address/network.rs

```diff
@@ -1,12 +1,18 @@
 // Copyright 2019-2022 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
+use num_enum::{FromPrimitive, IntoPrimitive};
```

Comment:
Please use `num_derive`, not `num_enum`.

Reply:
fixed